luzer/luzer.c

```diff
@@ -31,6 +31,10 @@
 #define CUSTOM_MUTATOR_LIB "libcustom_mutator.so.1"
 #define DEBUG_HOOK_FUNC "luzer_custom_hook"
 
+#define ENV_NOT_USE_CLI_ARGS "LUZER_NOT_USE_CLI_ARGS_FOR_LF"
```

Comment:
This is necessary so that if we fuzz functionality that works with command line arguments, there may be problems if the flags are the same.
For example:

``` lua
local function TestOneInput(buf)
    local fdp = luzer.FuzzedDataProvider(buf)
    local str = fdp:consume_string(4)

    local b = {}
    str:gsub(".", function(c) table.insert(b, c) end)
    if (b[1] == "o" and #arg > 0 and arg[1] == "-help=1") then assert(nil) end
end
```

if we need to set the "-help=1" flag, then without this environment variable, the fuzzer will simply launch its help and fuzzing will not even start, but with the variable set, this crash will be found.